packages/console/src/console-history/index.tsx

```diff
@@ -3,3 +3,4 @@ export { default as ConsoleHistoryItem } from './ConsoleHistoryItem';
 export { default as ConsoleHistoryItemResult } from './ConsoleHistoryItemResult';
 export { default as ConsoleHistoryResultErrorMessage } from './ConsoleHistoryResultErrorMessage';
 export { default as ConsoleHistoryResultInProgress } from './ConsoleHistoryResultInProgress';
+export { default as SizeUsageUI } from './SizeUsageUI';
```

Comment:
This isn't part of console history really. More part of status bar. The history is the part between status bar and input